packages/fxa-settings/src/components/ConnectedServices/Service.tsx

```diff
@@ -134,7 +134,7 @@ export function Service({
         <div className="flex flex-grow w-full mobileLandscape:justify-end mobileLandscape:flex-1">
           <Localized id="cs-sign-out-button">
             <button
-              className="cta-neutral cta-base disabled:cursor-wait whitespace-nowrap"
+              className="cta-neutral cta-base disabled:cursor-wait whitespace-nowrap py-2 px-5"
```

Comment:
Since `py-2 px-5` is what we almost always use, what do you think about creating a class for that? We could have:
```
.cta-base-p {
  @apply py-2 px-5;
}
```

Then if we ever want to update the "standard padding" for our buttons it's just in one spot, and it's easier to see what class is standard that goes with our cta classes.

Reply:
It looks like we're only using `tablet:px-10` on the `PageDeleteAccount` button (correct me if I'm wrong). That would be the special case that _doesn't_ get `cta-base-p`, and would be handled differently with padding utility classes. If we always adjusted the padding with media queries, then `cta-base-p` could also get a media query. That feels more maintainable to me since from a quick search I don't see that we're overriding that padding much. Though, if you still think it's better to leave as-is, I'll check into your branch later and poke around and likely r+. 😁

Reply:
Just because y'all got me curious about `tablet:*` specific stylings:
```sh
git grep -Eno "\btablet:[a-z0-9:\-]+\b" packages | sort | uniq
```
FILE | MATCH
:----|:----
fxa-react/components/Header/index.tsx:22 | `tablet:px-8`
fxa-react/components/LogoLockup/index.tsx:23 | `tablet:inline-flex`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:109 | `tablet:bg-violet-600`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:111 | `tablet:bg-violet-600`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:115 | `tablet:bg-transparent`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:115 | `tablet:text-white`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:118 | `tablet:bg-white`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:118 | `tablet:text-black`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:119 | `tablet:bg-violet-600`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:12 | `tablet:bg-green-400`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:13 | `tablet:bg-white`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:29 | `tablet:flex-row`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:38 | `tablet:px-1`
fxa-settings/.storybook/design-guide/pages/Breakpoints.tsx:41 | `tablet:px-1`
fxa-settings/.storybook/design-guide/pages/Colors.tsx:71 | `tablet:focus:bg-violet-300`
fxa-settings/.storybook/design-guide/pages/Colors.tsx:75 | `tablet:focus:bg-violet-300`
fxa-settings/.storybook/design-guide/pages/Colors.tsx:78 | `tablet:focus:bg-violet-300`
fxa-settings/src/components/AppLayout/index.tsx:23 | `tablet:px-20`
fxa-settings/src/components/BentoMenu/index.stories.tsx:11 | `tablet:pr-10`
fxa-settings/src/components/BentoMenu/index.tsx:59 | `tablet:relative`
fxa-settings/src/components/BentoMenu/index.tsx:59 | `tablet:w-auto`
fxa-settings/src/components/ConnectedServices/index.tsx:193 | `tablet:px-6`
fxa-settings/src/components/ConnectedServices/index.tsx:193 | `tablet:rounded-xl`
fxa-settings/src/components/DataCollection/index.tsx:61 | `tablet:px-6`
fxa-settings/src/components/DataCollection/index.tsx:61 | `tablet:rounded-xl`
fxa-settings/src/components/DataCollection/index.tsx:63 | `tablet:flex-7`
fxa-settings/src/components/DataCollection/index.tsx:63 | `tablet:ltr:pr-12`
fxa-settings/src/components/DataCollection/index.tsx:63 | `tablet:rtl:pl-12`
fxa-settings/src/components/DataCollection/index.tsx:85 | `tablet:justify-end`
fxa-settings/src/components/DataCollection/index.tsx:85 | `tablet:pr-4`
fxa-settings/src/components/DataCollection/index.tsx:85 | `tablet:pt-1`
fxa-settings/src/components/FlowContainer/index.tsx:30 | `tablet:my-10`
fxa-settings/src/components/FlowContainer/index.tsx:30 | `tablet:rounded-xl`
fxa-settings/src/components/FlowContainer/index.tsx:40 | `tablet:ltr:-ml-18`
fxa-settings/src/components/FlowContainer/index.tsx:40 | `tablet:ltr:mr-10`
fxa-settings/src/components/FlowContainer/index.tsx:40 | `tablet:rtl:-mr-18`
fxa-settings/src/components/FlowContainer/index.tsx:40 | `tablet:rtl:ml-10`
fxa-settings/src/components/LinkedAccounts/index.tsx:22 | `tablet:px-6`
fxa-settings/src/components/LinkedAccounts/index.tsx:22 | `tablet:rounded-xl`
fxa-settings/src/components/Modal/index.tsx:82 | `tablet:px-10`
fxa-settings/src/components/Nav/index.tsx:30 | `tablet:px-8`
fxa-settings/src/components/PageDeleteAccount/index.tsx:213 | `tablet:px-10`
fxa-settings/src/components/PageDeleteAccount/index.tsx:223 | `tablet:px-10`
fxa-settings/src/components/PageSettings/index.tsx:39 | `tablet:mx-0`
fxa-settings/src/components/Profile/index.tsx:18 | `tablet:rounded-xl`
fxa-settings/src/components/Security/index.tsx:39 | `tablet:rounded-xl`

Reply:
@LZoog I'm a little worried about relying on class components. In working on this they've felt very rigid and somewhat counterintuitive, and I think that might be because they're pushing back against Tailwind patterns -- but also this is new to me so I might just be adjusting still. Or did I misunderstand the suggestion?